deploy.env

```diff
@@ -0,0 +1,10 @@
+# ENV VARIABLES FOR DEPLOYMENT
+# You have to define the values in {}
+APP_NAME=melita-io
+ACCOUNT_NAME=weevenetwork
+DESCRIPTION="Decoder for Melita IO"
```

Comment:
Is it Decoder? 😄  This is a description for DockerHub repo, max 100 characters.

Reply:
updated it